google-ads-examples/src/main/java/com/google/ads/googleads/examples/extensions/AddSitelinks.java

```diff
@@ -0,0 +1,284 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.ads.googleads.examples.extensions;
+
+import com.beust.jcommander.Parameter;
+import com.google.ads.googleads.examples.utils.ArgumentNames;
+import com.google.ads.googleads.examples.utils.CodeSampleParams;
+import com.google.ads.googleads.lib.GoogleAdsClient;
+import com.google.ads.googleads.v2.common.AdScheduleInfo;
+import com.google.ads.googleads.v2.common.KeywordInfo;
+import com.google.ads.googleads.v2.common.SitelinkFeedItem;
+import com.google.ads.googleads.v2.enums.DayOfWeekEnum.DayOfWeek;
+import com.google.ads.googleads.v2.enums.ExtensionTypeEnum.ExtensionType;
+import com.google.ads.googleads.v2.enums.FeedItemTargetDeviceEnum.FeedItemTargetDevice;
+import com.google.ads.googleads.v2.enums.KeywordMatchTypeEnum.KeywordMatchType;
+import com.google.ads.googleads.v2.enums.MinuteOfHourEnum.MinuteOfHour;
+import com.google.ads.googleads.v2.errors.GoogleAdsError;
+import com.google.ads.googleads.v2.errors.GoogleAdsException;
+import com.google.ads.googleads.v2.resources.CampaignExtensionSetting;
+import com.google.ads.googleads.v2.resources.ExtensionFeedItem;
+import com.google.ads.googleads.v2.services.CampaignExtensionSettingOperation;
+import com.google.ads.googleads.v2.services.CampaignExtensionSettingServiceClient;
+import com.google.ads.googleads.v2.services.ExtensionFeedItemOperation;
+import com.google.ads.googleads.v2.services.ExtensionFeedItemServiceClient;
+import com.google.ads.googleads.v2.services.MutateCampaignExtensionSettingResult;
+import com.google.ads.googleads.v2.services.MutateCampaignExtensionSettingsResponse;
+import com.google.ads.googleads.v2.services.MutateExtensionFeedItemResult;
+import com.google.ads.googleads.v2.services.MutateExtensionFeedItemsResponse;
+import com.google.ads.googleads.v2.utils.ResourceNames;
+import com.google.common.collect.ImmutableList;
+import org.joda.time.DateTime;
+import com.google.protobuf.Int32Value;
+import com.google.protobuf.StringValue;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+/** Adds Sitelinks to a campaign. To create a campaign, run AddCampaigns.java. */
```

Comment:
Nit: We don't capitalize sitelinks. It's just a plain lowercase. One good way to check capitalization is to verify in the Help Center such as this page: https://support.google.com/google-ads/answer/2375416?hl=en
Could you please change this?